scss/_theme.scss

```diff
@@ -154,16 +154,22 @@ h4 {
   .panel-heading,
   .panel-title {
     font-weight: 400;
-    padding: 8px
-
     a {
       display: block;
       cursor: pointer;
     }
+    a:after {
+      font-family: 'Glyphicons Halflings';  /* essential for enabling glyphicon */
+      content:"\e259";
+      float: right;
+    }
+    a.collapsed:after {
+      content:"\e257";
```

Comment:
It would be nice to do an animated transform here, something like 

```
transform: rotate(90deg);
// then some animation duration stuff
```

Reply:
@ianks I dont think it really needs an animation having run it locally. and yes it only applies to collapsible panels